### ISSUE TITLE
[studio-ui] 2.5 bulk go live says "Bulk go live failed" when REST call times out (http conn timeout)

### ISSUE THREAD

**Issue by russdanner** [craftercms/craftercms#792]
Currently, the Bulk Go Live UI says "bulk go live failed" when the request times out.  This is not accurate -- the request is still going on the server.

In the timeout scenario:
Can we report: 'The bulk publish operation is still be running.'

**Comment by vestrada** [comment]
studio-ui - Master - PR: https://github.com/craftercms/studio2-ui/pull/396
studio-ui - 2.5.x - PR: https://github.com/craftercms/studio2-ui/pull/397

**Comment by vestrada** [comment]
Per our conversation we are returning the message immediately and telling the user the publish process has been started.

**Comment by sumerjabri** [comment]
@russdanner Please test. If it passes, please close and assign to me.

**Comment by russdanner** [comment]
I don't understand the implementation here.  We only report that we've started the publish but the user has no cue at all about what's happening -- more importantly what to expect

Is it at all possible (http://yui.github.io/yui2/docs/yui_2.9.0_full/connection/#timeout) to do:
ON FIRE: Process has started
ON SUCCESS: "Bulk publish process has been started."
ON FAILURE: "Bulk publish process has failed: {ERROR}.
ON TIMEOUT: "Bulk publish monitoring has timed out.  Timeouts are often due to the amount of content published.  Publishing will continue although monitoring has ceased."

I apologize I don't remember if we discussed a specific limitation.
